anesthetic/samples.py

```diff
@@ -1334,6 +1372,168 @@ def recompute(self, logL_birth=None, inplace=False):
         else:
             return samples.__finalize__(self, "recompute")
 
+    def plot_types(self):
+        """
+        Get types of plots supported by this class.
+
+        Returns
+        -------
+        tuple[str]
+        """
+        return 'live', 'posterior'
```

Comment:
In the bigger scheme of anesthetic, this is misleading. We have used the kwargs `types` and `plot_type` in the past to indicate KDE plots, histograms, scatter plots, etc. These kwargs have been renamed to `kind`, to unify with naming conventions in pandas and matplotlib.

I don't think this should be a method of `NestedSamples`. This seems to be a more GUI specific thing, so a simple list there makes probably more sense.